# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/NotificationParent.cpp
# Commit: 93937caf6d13
# Full Hash: 93937caf6d13cd0d6ef427f3315e6a29c30a385c
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-08 04:19:40
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 1: Add skeleton PNotification r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227686
# ==============================================================================

diff -r 785541f0311a -r 93937caf6d13 dom/notification/NotificationParent.cpp
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/notification/NotificationParent.cpp	Thu Nov 07 17:32:03 2024 +0000
@@ -0,0 +1,23 @@
+/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
+/* vim: set ts=8 sts=2 et sw=2 tw=80: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "NotificationParent.h"
+
+namespace mozilla::dom::notification {
+
+NS_IMPL_ISUPPORTS(NotificationParent, nsIObserver)
+
+NS_IMETHODIMP
+NotificationParent::Observe(nsISupports* aSubject, const char* aTopic,
+                            const char16_t* aData) {
+  return NS_OK;
+}
+
+mozilla::ipc::IPCResult NotificationParent::RecvShow(ShowResolver&& aResolver) {
+  return IPC_OK();
+}
+
+}  // namespace mozilla::dom::notification